crates/cext/src/circuit.rs

```diff
@@ -995,7 +1121,7 @@ pub unsafe extern "C" fn qk_circuit_instruction_clear(inst: *mut CInstruction) {
         inst.num_clbits = 0;
         if inst.num_params > 0 && !inst.params.is_null() {
             let params = std::slice::from_raw_parts_mut(inst.params, inst.num_params as usize);
-            let _ = Box::from_raw(params as *mut [f64]);
+            let _ = Box::from_raw(params);
             inst.params = std::ptr::null_mut();
```

Comment:
This doesn't drop the `Param` objects, just the pointers to them.  It worked for `f64` because it doesn't have a `Drop` impl, but `Param` does and needs to be dropped correctly.